### PR TITLE
issue #62, add legend with bin labels to plotMotifHeatmaps

### PR DESCRIPTION
**Pull request by danymukesha**

### CODE REVIEW

File: R/plotting.R [hunk 1]
```diff
@@ -1,734 +1,737 @@
...
+#' @param show_dendrogram If \code{cluster != FALSE}, controls whether to show
+#'     a row dendrogram for the clustering of motifs. Ignored for 
+#'     \code{cluster = FALSE}.
+#' @param show_motif_GC If \code{TRUE}, show a column with the percent G+C of 
+#'     the motif as part of the heatmap.
+#' @param show_seqlogo If \code{TRUE}, show a sequence logo next to each motif 
+#'     label. This will likely only make sense for a heatmap with a low number 
+#'     of motifs.
+#' @param show_bin_legend If \code{TRUE}, show a legend for the bin labels.
+#'     If FALSE (default), the bin legend will be hidden .
```
Comment (mbstadler) [inline R/plotting.R]: there is a unnecessary whitespace before the full stop

**Comment by mbstadler** [comment]
@danymukesha this looks great, thanks a lot!

Could you add the following:

- at the beginning of the function, use `.assertScalar(...)` to make sure that the argument is indeed a logical scalar
- to the unit test file (probably to the existing calls of the function), add a case for each `show_bin_legend=TRUE` and `show_bin_legend=FALSE`, to make sure it is tested with both values

Once this is added, we'll merge it, bump the version, add you as a contributor and push to bioconductor

**Comment by danymukesha** [comment]
Thank you @mbstadler. 
You are right. I think it is a good practice to assert if the argument `show_bin_legend` is a logical scalar.

**Comment by mbstadler** [comment]
Thank you @danymukesha , this looks all good. I'll go forward with merging and additions.

**Comment by mbstadler** [comment]
It's pushed upstream, should become available once the `JASPAR2024` gets resolved (monaLisa version 1.11.3)
